lib/email.js

```diff
@@ -215,6 +213,8 @@ Email.prototype.send = function send (callback) {
   };
   var signedOpts = aws4.sign(options, credentials);
 
+  signedOpts.headers['Connection'] = 'keep-alive';
```

Comment:
The AWS signing algorithm does not play well with the `Connection` header. Adding it after signing will still apply it to the request, but does not break the signing algorithm.

This does not affect all regions. Looks like the signing algorithm implementation varies between regions.